website_seo_redirection/tests/test_ui.py

```diff
@@ -2,15 +2,20 @@
 # © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
-from openerp.tests.common import HttpCase
+from odoo.tests.common import HttpCase
 
 
 class UICase(HttpCase):
-    def test_admin_tour_en_US(self):
+
+    post_install = True
+    at_install = False
+
+    def test_admin_tour(self):
         """Redirections work with default language."""
-        tour = "odoo.__DEBUG__.services['web.Tour']"
+        tour = "odoo.__DEBUG__.services['web_tour.tour']"
         self.phantom_js(
-            "/en_US",
```

Comment:
I think I prefer if you leave this URL, just because it should store a cookie and trigger a redirection by itself to `/`, which was a special case this addon had to take into account if I don't remember badly... Is it possible?